content/pages/framework-guides/deploy-a-nextjs-site.md

```diff
@@ -7,7 +7,162 @@ title: Deploy a Next.js site
+
+For more information about the Edge Runtime, you can check out [the official Next.js documentation](https://nextjs.org/docs/advanced-features/react-18/switchable-runtime) which elaborates on the differences between the Edge Runtime and traditional Node.js servers, or you can [read our announcement blog post](https://blog.cloudflare.com/next-on-pages).
+
+{{<render file="_tutorials-before-you-start.md">}}
+
+## Creating a new project using the Edge Runtime
+
+Create a new project using `npx` or `yarn` by running either of the following commands in your terminal:
+
+```sh
+$ npx create-next-app my-app
+# or
+$ yarn create next-app my-app
+```
+
+After creating your project, a new `my-app` directory will be generated using the official default template.
+
+### Configuring the project to use the Edge Runtime
+
+The default template includes a traditional Node.js-powered API route (`pages/api/hello.js`). This is incompatible with the Edge Runtime, so delete this file. If you wish to use API routes, check out the documentation on Next.js' new [Edge API Routes](https://nextjs.org/docs/api-routes/edge-api-routes), which offers a standards-based equivalent.
+
+As an example, the existing `pages/api/hello.js` file re-written as an Edge API Route would look like this:
+
+{{<tabs labels="js | ts">}}
+{{<tab label="js" default="true">}}
+
+```js
+---
+filename: pages/api/hello.js
+---
+// Next.js Edge API Routes: https://nextjs.org/docs/api-routes/edge-api-routes
+
+export const config = {
+  runtime: 'experimental-edge',
+}
+
+export default async function (req) {
+  return new Response(
+    JSON.stringify({ name: 'John Doe' }),
+    {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }
+  )
+}
+```
+
+{{</tab>}}
+{{<tab label="ts" default="true">}}
+
+```ts
+---
+filename: pages/api/hello.ts
+---
+// Next.js Edge API Routes: https://nextjs.org/docs/api-routes/edge-api-routes
+
+import type { NextRequest } from 'next/server'
+
+export const config = {
+  runtime: 'experimental-edge',
+}
+
+export default async function (req: NextRequest) {
+  return new Response(
+    JSON.stringify({ name: 'John Doe' }),
+    {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }
+  )
+}
+```
+
+{{</tab>}}
+{{</tabs>}}
+
+Next, we need to configure the rest of the project to use the Edge Runtime. This can be done globally by adding the following to your `next.config.js` file:
```

Comment:
```suggestion
Next, you must configure the rest of the project to use the Edge Runtime. This can be done globally by adding the following to your `next.config.js` file:
```